### PR TITLE
Burak mtsk 471 contacts

### PR DESCRIPTION
**Pull request by ARIKBOGA**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -81,12 +81,6 @@
             <version>23.0.0</version>
             <scope>test</scope>
         </dependency>
-
```
Comment (zeroist) [inline pom.xml:84]: do you really want to delete javafaker dependency

File: src/test/java/com/meetsky/step_definitions/Contacts_StepDefinition.java [hunk 2]
```diff
@@ -170,7 +170,7 @@ public void userShouldBeAbleToSeeNewCreatedContact() {
     @And("User fills these properties out")
     public void userFillsThesePropertiesOut(DataTable table) {
         Map<String, String> inputMap = table.asMap();
-        selectUnselectedInputs();
```
Comment (zeroist) [inline src/test/java/com/meetsky/step_definitions/Contacts_StepDefinition.java:173]: did you delete the class since you use import java.util.*;

**Review by zeroist — COMMENTED** [review]
after I make sure that you are sure I will approve

**Review by ARIKBOGA — COMMENTED** [review]
viewed

**Review by ARIKBOGA — COMMENTED** [review]
submitted

**Review by ARIKBOGA — COMMENTED** [review]
submitted

**Review by ARIKBOGA — COMMENTED** [review]
submitted

**Pull request closed without merging**